lambdas/build-proven-user-identity-details/src/main/java/uk/gov/di/ipv/core/buildprovenuseridentitydetails/BuildProvenUserIdentityDetailsHandler.java

```diff
@@ -140,11 +140,14 @@ public Map<String, Object> handleRequest(JourneyRequest input, Context context)
         }
     }
 
-    private Map<String, Object> buildJourneyErrorResponse(ErrorResponse errorResponse) {
+    private APIGatewayProxyResponseEvent buildJourneyErrorResponse(ErrorResponse errorResponse) {
+        return buildJourneyErrorResponse(errorResponse, HttpStatus.SC_INTERNAL_SERVER_ERROR);
+    }
+
+    private APIGatewayProxyResponseEvent buildJourneyErrorResponse(
+            ErrorResponse errorResponse, int statusCode) {
         LOGGER.error(LogHelper.buildLogMessage(errorResponse.getMessage()));
-        return new JourneyErrorResponse(
-                        JOURNEY_ERROR_PATH, HttpStatus.SC_INTERNAL_SERVER_ERROR, errorResponse)
-                .toObjectMap();
+        return ApiGatewayResponseGenerator.proxyJsonResponse(statusCode, errorResponse);
```

Comment:
This is a small change, so error responses from this endpoint no longer look like journey events.

old:
```
{
  "journey": "/journey/error",
  "statusCode": 500,
  "code": 1234,
  "message": "an error happened"
}
```

new:
```
{
  "code": 1234,
  "message": "an error happened"
}
```